lib/middleman-hashicorp/partials/_enterprise-alert.html.erb

```diff
@@ -0,0 +1,12 @@
+<notextile>
```

Comment:
It seems sensical to keep the snippet in html so it can be pulled into Markdown and non-markdown files, but pulling the partial into markdown files as straight html was causing some unexpected behavior, most notably empty <p> tags. I did a little research but the usual curprit of that issue, incorrect html, didn't seem to be the cause. It checked out. <notextile> prevents the html inside it from being parsed as markdown, and once I added that the problem seemed to be resolved. I'm definitely open to other solutions, though.